### DISCUSSION TITLE
How do i increase compilation time for my tailwind css? Its taking up to 6secs to compile Im using @apply and applying the styles to className.

### DISCUSSION THREAD

**Discussion by Tosinkoa** [tailwindlabs/tailwindcss#6660] [Help]
Compilation takes so much time, it takes up to 6secs, can someone please tell me what to do?

**Comment by adamwathan** [comment]
Can you share your project? Not possible to help without understanding how you have things set up.

**Reply by Tosinkoa** [reply]
Thanks for replying sir, i'm still learning right now and I'm sorry to ask this if it might sound dumb cuz I have to be honest with my standard.
Do you mean I should share the repository link, or share my code....?

**Reply by Tosinkoa** [reply]
Here is the link to the project:
https://github.com/Tosinkoa/My-Gallery-Project.git
